src/resources/postcss/views/_list.pcss

```diff
@@ -0,0 +1,113 @@
+/**
+ * The Events Calendar Tribe List View V2 Stylesheet
+ *
+ * @package TribeEventsCalendar
+ *
+ */
+.tribe-events {
```

Comment:
Is there a reason why we shouldn't use `tribe-events` here @bordoni ? We defined this in the CSS planning document and it would be good if we didn't have to switch it up here. It also helps us follow BEM format.